pkg/app/execontext.go

```diff
@@ -120,7 +133,17 @@ func (ref *Output) Message(data string) {
 	color.Set(color.FgHiMagenta)
 	defer color.Unset()
 
-	fmt.Printf("cmd=%s message='%s'\n", ref.CmdName, data)
+	if ref.JSONFlag == "json" {
+		//marshal data to json
+		var jsonData []byte
```

Comment:
same here...

we need a map with the `cmd` and `message` fields, which we can then json marshal, etc

Reply:
Done